packages/ui/src/utils/sign-in-experience.ts

```diff
@@ -27,21 +27,16 @@ const getSecondarySignInMethod = (signInMethods: SignInMethods) =>
     return methods;
   }, []);
 
-const getSignInExperienceSettings = async () => {
-  try {
-    const result = await getSignInExperience();
-    const settings = {
-      branding: result.branding,
-      languageInfo: result.languageInfo,
-      termsOfUse: result.termsOfUse,
-      primarySignInMethod: getPrimarySignInMethod(result.signInMethods),
-      secondarySignInMethods: getSecondarySignInMethod(result.signInMethods),
-    };
-
-    return { settings };
-  } catch (error: unknown) {
-    return { error };
-  }
```

Comment:
Does the API not throw error any more, or do we have error handling somewhere else?

Reply:
just follow GG's last comment, let this API throw, and break the page, as technically this result is a hard blocker. Will discuss a proper user experience later if needed.

Reply:
ok